gradle/libs.versions.toml

```diff
@@ -55,7 +55,7 @@ commons-collections3 = "3.2.2"
 commons-configuration1 = "1.6"
 commons-dbcp2 = "2.11.0"
 caffeine = "2.9.3"
-iceberg = '1.5.2' # used for Gravitino Iceberg catalog and Iceberg REST service
+iceberg = '1.6.1' # used for Gravitino Iceberg catalog and Iceberg REST service
```

Comment:
Do U need to modify the playground, too? Maybe you can create an issue in the repo of the playground.

Reply:
Yes, we should modify more places,  just check whether Iceberg `1.6.1` works on Gravitino for now.

Reply:
create  https://github.com/apache/gravitino-playground/issues/129 to track it

Reply:
OK.